_overviews/scala3-book/domain-modeling-tools.md

```diff
@@ -33,27 +33,27 @@ class Movie(var name: String, var director: String, var year: Int)
 ```
 
 These examples show that Scala has a very lightweight way to declare classes.
-The definition of the class `Person` roughly corresponds to the following, more explicit, version:
+The definition of the class `Person` corresponds to the following Java version:
 
-```scala
-class Person:
+```java
+class Person {
   // fields
-  var name: String = null
-  var vocation: String = null
-
+  String name;
+  String vocation;
+  
   // constructor
-  def this(_name: String, _vocation: String) =
-    // call to the super constructor
-    this()
+  public Person(String name, String vocation) {
     // assigning the fields
-    name = _name
-    vocation = _vocation
+    this.name = name;
+    this.vocation = vocation;
+  }
+}
 ```
```

Comment:
The original Scala code was not valid. My commit uses a Java version, which makes sense only to Java users, so I’m not sure it’s a good idea, maybe we should just remove that code snippet?

Reply:
I’m sorry about that initial Scala code, that definitely wasn’t the right approach.

I can go either way on this. On the one hand, just showing the Scala code shows how concise it is. People who use any other language can make the “conciseness” comparison in their head.

My original logic was (a) I assume that most developers have worked with Java, and (b) I wanted to appeal to Java developers. But again, they can make that comparison in their head. (And if we show Java, why not show Python as well?)

So yes, after thinking about it, I would delete that comparison to Java.

Reply:
OK, thanks for sharing your thoughts. I have removed the Java code snippet.